content/articles/azure-devops-api-development/index.md

```diff
@@ -0,0 +1,127 @@
+configuration.
+
+![intro](/engineering-education/azure-devops-api-development/intro.png)
+
+### Step 1: Creating Resource Group in Azure
+
+Once you have a subscription in Azure, you need to create a `resource group` to manage resources in Azure. The best practice is to maintain a separate resource group for each environment (DEV, INT, QA, STAG, PERF, PROD) and follow the proper naming conventions. To create a resource group, select Resource Group from the left-side blade, `click on +Add`, then select `Subscription`, give a proper name to the resource group, and click on `Review + Create`. Refer to the image below.
+
+![resource-group](/engineering-education/azure-devops-api-development/resource-group.jpg)
+
+### Step 2: Creating App Service
+
+Click on `All resources` from the left-side blade of the panel and click on `Add`, then select `Web App` and give a proper name to your app service. Then select `Subscription` and select the existing resource group. Keep the remaining option as default. and click on `Create`.
+
+![creating-app-service](/engineering-education/azure-devops-api-development/creating-app-service.jpg)
+
+Once you have created the app service, click on that app service and go to `Overview`. There you can identify all the details that belong to this app service, including status, resource group, subscription, and URL (please make a note of this; you’ll need it later).
+
+![overview](/engineering-education/azure-devops-api-development/overview.jpg)
+
+### Step 3: Creating Service Principal
+
+When an application needs access to deploy or configure resources through ARM or VSTS in Azure, you’ll need to create a service principal, which is a credential for your application. Go to Azure `Active Directory -> App registrations -> New application registration`, then give a name to the service principle, select application type and give it a URL of your choice. After creating the app registration, go to `Settings` from that service and make note of application ID, then go to `Keys` create a key and copy the secret value. (Note: It doesn’t appear later, so please store it in a safe place.)
+
+![service-principal](/engineering-education/azure-devops-api-development/service-principal.jpg)
+
+![settings-keys](/engineering-education/azure-devops-api-development/settings-keys.jpg)
+
+### Step 4: Create a New Project in VSTS
+
+The first step is to create an account in Azure DevOps (VSTS), then follow the steps below to start an application deployment. Here we will take a [sample open-source java-based code](https://github.com/wakaleo/game-of-life) from GitHub repository and import it to Azure DevOps repo. (You can push your source code into the repo.) It is a simple multi-module Maven project. The application is a very simple online version of Conway’s `Game of Life.
+

```

Comment:
You forgot the closing tag in Game of Life

Reply:
@WanjaMIKE done